src/main/java/com/gmail/goosius/siegewar/playeractions/PlunderTown.java

```diff
@@ -50,18 +50,18 @@ private static Nation getPlunderingNationOrThrow(Player player, Siege siege) thr
 		final Translator translator = Translator.locale(player);
 
 		if (!townyUniverse.getPermissionSource().testPermission(player, SiegeWarPermissionNodes.SIEGEWAR_NATION_SIEGE_PLUNDER.getNode()))
-			throw new TownyException(translator.of("msg_err_command_disable"));
+			throw new TownyException(translator.of("msg_err_cannot_plunder_not_enough_permissions"));
 
 		Resident resident = townyUniverse.getResident(player.getUniqueId());
 		if (resident == null)
 			throw new TownyException(translator.of("msg_err_not_registered_1", player.getName()));
 
 		if(!resident.hasTown())
-			throw new TownyException(translator.of("msg_err_siege_war_action_not_a_town_member"));
+			throw new TownyException(translator.of("msg_err_cannot_plunder_not_enough_permissions"));
 
 		Nation plunderingNation = resident.getNationOrNull();
 		if(plunderingNation == null)
-			throw new TownyException(translator.of("msg_err_siege_war_action_not_a_nation_member"));
+			throw new TownyException(translator.of("msg_err_cannot_plunder_not_enough_permissions"));
```

Comment:
```suggestion
			throw new TownyException(translator.of(""msg_err_dont_belong_nation""));
```

Reply:
Resolved. The line just above this one now checks the nation and gives an appropriate error message.